scripts/install-hooks.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+PRE_COMMIT_FILE="$(dirname -- "${BASH_SOURCE[0]}")/../.git/hooks/pre-commit"
+
+cat <<'EOF' >$PRE_COMMIT_FILE
+#!/bin/bash
```

Comment:
Unless for certain reasons, I like putting this at the top of the bash scripts (after the shebang) to prevent unwanted behaviors:
```
set -eu -o pipefail
```

Quoting `set --help`:
```
-e  Termine immédiatement si une commande s'arrête avec un code de retour non nul.
        -o nom-option
            Défini la variable correspondant à nom-option :
                ....
                pipefail     le code de retour d'un tube est celui de la dernière commande
                             qui s'est terminée avec un code non nul,
                             ou zéro si aucune commande ne s'est arrêtée avec un code non nul.
-u  Traite les variables non définies comme des erreurs lors de la substitution.